queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Factories/SignatureItemFactoryDEFAULT.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.Middleware.Contracts.Factories;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Factories
+{
+    public class SignatureItemFactoryDEFAULT : SignatureItemFactory
+    {
+        public override long CountryBaseState => 000000000;
+
+        public SignatureItemFactoryDEFAULT() { 
+        }
+
+        protected static NumberFormatInfo CurrencyFormatter = new ()
+        {
+            NumberDecimalSeparator = ",",
+            NumberGroupSeparator = "",
+            CurrencyDecimalDigits = 2
+        };
+
+        public SignaturItem[] CreateSignatures(DailyClosingResponse dailyClosingResponse) //CreateSignatures then remove the other functions
```

Comment:
Should we add some signatures here like SANDBOX and such? Maybe also a QR Code with www.fiskaltrust.eu. So at least those signatures that we append to the country specific once can also be added :)

Reply:
The `S A N D B O X` signature is added by the `Queue` https://github.com/fiskaltrust/middleware/blob/main/queue/src/fiskaltrust.Middleware.Queue/SignatureFactory.cs. A default QR code is a nice idea 🙏